sample-lnd.conf

```diff
@@ -438,6 +438,12 @@
 ; If true, all HTLCs will be held until they are handled by an interceptor
 ; requireinterceptor=false
 
+; If true, lnd will also accept positive inbound fees. By default, lnd only
+; accepts negative inbound fees (an inbound "discount") to keep the channels
```

Comment:
:s/`to keep the channels open for senders that do not recognize inbound fees`/`to remain backwards compatible with older clients but also clients which do not support inbound fees yet (CLN, LNDK, Eclair). Therefore you should ONLY set this setting if you know what you are doing`.

Reply:
included